docs/scos/dev/glue-api-guides/202204.0/glue-spryks.md

```diff
@@ -104,8 +106,8 @@ For more details, see section **Controller management** in [Glue Infrastructur
 
 | SPRYK | DESCRIPTION |
 | --- | --- |
-| `AddGlueControllerBeforeActionPlugin` | Adds a new **before action** plugin to the given business model of the given module.<br>**Arguments**<p>**General Arguments:**</p><ul><li>**mode** - *project* or *core*.<br />The default is **project**. For details, see [Difference between Core and Project modes](/docs/sdk/dev/spryks/spryks.html#core-and-project-modes).</li><li>**organization** - specifies the name of the organization for which to perform the operation.<br />The default is **Pyz**.</li><li>**module** - specifies the module name.</li></ul><p>**Spryk-Specific Arguments:**</p>**modelName** - specifies the model name.<br />It will be used as a prefix for the plugin name. For example, if you specify `My` for *modelName*, the resulting plugin name will be `SetMyControllerBeforeActionPlugin`. |
-| `AddGlueControllerAfterActionPlugin` | Adds a new **after action** plugin to the given business model of the given module.<br>**Arguments**<p>**General Arguments:**</p><ul><li>**mode** - *project* or *core*.<br />The default is **project**. For details, see [Difference between Core and Project modes](/docs/sdk/dev/spryks/spryks.html#core-and-project-modes).</li><li>**organization** - specifies the name of the organization for which to perform the operation.<br />The default is **Pyz**.</li><li>**module** - specifies the module name.</li></ul><p>**Spryk-Specific Arguments:**</p>**modelName** - specifies the model name.<br />It will be used as a prefix for the plugin name. For example, if you specify `My` for *modelName*, the resulting plugin name will be `SetMyControllerAfterActionPlugin`. |
+| `AddGlueControllerBeforeActionPlugin` | Adds a new **before action** plugin to the given business model of the given module.<br>**Arguments**<p>**General Arguments:**</p><ul><li>**mode** - *project* or *core*.<br />The default is **project**. For details, see [Difference between Core and Project modes](/docs/sdk/dev/spryks/spryks.html#core-and-project-modes).</li><li>**organization** - specifies the name of the organization for which to perform the operation.<br />The default is **Pyz**.</li><li>**module** - specifies the module name.</li><li>**applicationType** - specifies which application type should be used: Backend or Storefront. By default the **Backend** application type is used.</li></ul><p>**Spryk-Specific Arguments:**</p>**modelName** - specifies the model name.<br />It will be used as a prefix for the plugin name. For example, if you specify `My` for *modelName*, the resulting plugin name will be `SetMyControllerBeforeActionPlugin`. |
```

Comment:
```suggestion
| `AddGlueControllerBeforeActionPlugin` | Adds a new **before action** plugin to the given business model of the given module.<br>**Arguments**<p>**General Arguments:**</p><ul><li>**mode** - *project* or *core*.<br />The default is **project**. For details, see [Difference between Core and Project modes](/docs/sdk/dev/spryks/spryks.html#core-and-project-modes).</li><li>**organization** - specifies the name of the organization for which to perform the operation.<br />The default is **Pyz**.</li><li>**module** - specifies the name of the module.</li><li>**applicationType** - specifies which application type should be used: Backend or Storefront. By default the **Backend** application type is used.</li></ul><p>**Spryk-Specific Arguments:**</p>**modelName** - specifies the model name.<br />It will be used as a prefix for the plugin name. For example, if you specify `My` for *modelName*, the resulting plugin name will be `SetMyControllerBeforeActionPlugin`. |
```